src/main/java/io/jenkins/plugins/analysis/core/model/DetailsTableModel.java

```diff
@@ -31,115 +32,88 @@
  *
  * @author Ullrich Hafner
  */
-public abstract class DetailsTableModel {
+public abstract class DetailsTableModel extends TableModel {
     private final AgeBuilder ageBuilder;
     private final FileNameRenderer fileNameRenderer;
     private final DescriptionProvider descriptionProvider;
+    private final Report report;
 
     /**
      * Creates a new instance of {@link DetailsTableModel}.
      *
-     * @param ageBuilder
-     *         renders the age column
+     * @param report
+     *         the report to render
      * @param fileNameRenderer
      *         renders the file name column
+     * @param ageBuilder
+     *         renders the age column
      * @param descriptionProvider
      *         renders the description text
      */
-    protected DetailsTableModel(final AgeBuilder ageBuilder, final FileNameRenderer fileNameRenderer,
+    protected DetailsTableModel(final Report report, final FileNameRenderer fileNameRenderer,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [It is a good practice to call super() in a constructor](https://app.codacy.com/app/uhafner/warnings-ng-plugin/pullRequest?prid=4437947)